docs/guides/basic-concepts.md

```diff
@@ -66,6 +66,19 @@ Example:
 const { value, error, touched, isValidating } = field.state
 ```
 
+There are three field states can be very useful to see how the user interacts with a field. A field is _"touched"_ when the user clicks/tabs into it, _"pristine"_ until the user changes value in it, and _"dirty"_ after the value has been changed. You can check these states via the `isTouched`, `isPristine` and `isDirty` flags, as seen below.
+
+```tsx
+const { isTouched, isPristine, isDirty } = field.state.meta
+```
+
+![Field states](https://github.com/TanStack/form/assets/43729152/513c29d4-23bc-4934-9658-9a5856ad2ef6)
```

Comment:
Let's upload this image inside of `assets` on the repo or something. That way, GH can't accidentally GC it and we can keep track of it locally

Reply:
Ugh, that'd be annoying! 😆  I uploaded it to `/docs/assets/field-states.png`, and updated the link.